Confus/Confus/Game.cpp

```diff
@@ -54,8 +55,18 @@ namespace Confus
 			videoDriver->getTexture("Media/Textures/Heart.png"),
 			videoDriver->getTexture("Media/Textures/Blood.png"),
 			irr::core::vector2df(0.0f, 0.02f));
+        try
```

Comment:
Because I'm throwing an exception when the image can't be found. (And if I'm not catching this here it will throw a read access violation later on)